server/server.py

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+from pathlib import Path
+
+from flask import Flask, request
+from flask.json import jsonify
+from werkzeug.utils import secure_filename
+
+
+URLS = []
+ALLOWED_EXTENSIONS = set(["txt", "pdf", "png", "jpg", "jpeg", "gif", "json"])
+
+Path("files").mkdir(exist_ok=True)
+app = Flask("klaatu_server")
+app.secret_key = "secret key"
+app.config["UPLOAD_FOLDER"] = "files"
+app.config["MAX_CONTENT_LENGTH"] = 16 * 1024 * 1024
+
+
+def allowed_file(filename):
+    return "." in filename and filename.rsplit(".", 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+@app.route("/test_results", methods=["POST"])
+def test_results():
+    request_file = request.files["file"]
+    if request_file and allowed_file(request_file.filename):
+        filename = secure_filename(request_file.filename)
+        request_file.save(os.path.join(app.config["UPLOAD_FOLDER"], filename))
```

Comment:
Not sure if this server has auth or anything else, are we worries about random people uploading files to the server? Or is Klaatu service short lived for the duration of a test and a reasonably small risk of this being abused. (not that I expect anybody to try load testing file uploading w/ 15.9MB PDF files and exhausting all server resources).

Reply:
Hmm, good point. I need to think about this

Reply:
Looking into it, the ```MAC_CONTENT_LENGTH``` setting should be able to handle this fairly well.